### PR TITLE
Video nav test

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What does this change?

Adding a test to see what happens when we add video to the main nav.
Does it cannibalise other clicks, or just interest people in video.

We will be making the video front a little better before sending this out.
## What is the value of this and can you measure success?

People want video more, but not at the expense of others.

This isn't really an A/B per-say - more a test about, if we did this, what would the affect be? We can then make more informed decisions around it later. 
## Does this affect other platforms - Amp, Apps, etc?

No
## Screenshots

video-nav
## Request for comment

@akash1810 @harrysalmon

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/video-nav.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+
+    return function () {
+        this.id = 'VideoNav';
+        this.start = '2016-06-06';
+        this.expiry = '2016-06-13';
+        this.author = 'James Gorrie';
+        this.description = 'Use video carousel as video nav';
+        this.audience = .2;
```
Comment (harrysalmon) [inline static/src/javascripts/projects/common/modules/experiments/tests/video-nav.js]: You mentioned changing this

File: static/src/javascripts/projects/common/modules/experiments/tests/video-nav.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+define([], function () {
+    function setVideoOnNav(text) {
+        var videoIcon = document.querySelector('.inline-video-icon').cloneNode(true);
```
Comment (akash1810) [inline static/src/javascripts/projects/common/modules/experiments/tests/video-nav.js]: will this work if there's no video on the front?

File: static/src/javascripts/projects/common/modules/experiments/tests/video-nav.js (hunk 1 above)
Comment (jamesgorrie) [inline static/src/javascripts/projects/common/modules/experiments/tests/video-nav.js]: Looking at the a/b dashboard - we're good between 10 & 12%?

File: static/src/javascripts/projects/common/modules/experiments/tests/video-nav.js (hunk 2 above)
Comment (jamesgorrie) [inline static/src/javascripts/projects/common/modules/experiments/tests/video-nav.js]: Good catch, changed.

File: static/src/stylesheets/module/nav/_nav.scss [hunk 3]
```diff
@@ -119,3 +119,12 @@
         }
     }
 }
+
+.video-nav-test-icon {
+    svg {
+        //fill: #ffbb00;
```
Comment (akash1810) [inline static/src/stylesheets/module/nav/_nav.scss]: not needed?

**Comment by akash1810** [comment]
👍

**Pull request merged**